src/main/java/uk/gov/hmcts/ccd/domain/service/caseaccess/CaseAccessOperation.java

```diff
@@ -30,15 +30,15 @@ public void grantAccess(final String jurisdictionId, final String caseReference,
                                                                                       Long.valueOf(caseReference));
 
         final CaseDetails caseDetails = maybeCase.orElseThrow(() -> new CaseNotFoundException(caseReference));
-        caseUserRepository.grantAccess(caseDetails.getId(), userId);
+        caseUserRepository.grantAccess(Long.valueOf(caseDetails.getId()), userId);
```

Comment:
Now that we changed caseId to String, I think we should raise a tech debt story convert all our methods that still expect caseId to be a Long. Otherwise we'll have to add these conversions everywhere

Reply:
https://tools.hmcts.net/jira/browse/RDM-2695

Reply:
good point